lib/app/models/testing_instance.rb

```diff
@@ -52,6 +52,12 @@ class TestingInstance
 
       property :dynamic_registration_token, String
 
+      property :practitioner, String
+      property :practitionerrole, String
+      property :medication, String
+      property :organization, String
+      property :location, String
```

Comment:
Could there be a more general solution here? Manually maintaining a list of special resources in `TestingInstance` and the database schema seems problematic.

Reply:
Made it so the properties are added in the sequences.